src/Microsoft.NET.Test.Sdk.props

```diff
@@ -11,8 +11,10 @@ Copyright (c) .NET Foundation. All rights reserved.
 -->
 
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <ProjectCapability Include="TestContainer" />
```

Comment:
I think you should be putting this in the packages themselves, I can install xUnit in a random project that doesn't have to be test project (in fact Brad mentioned they are enabling that in their new dotnet xunit runner).

We should get xUnit, NUnit, etc to all mark themselves as a TestContainer